### PR TITLE
Loki Merge Upstream #e8487fa 2019-05-17

### PR DESCRIPTION
**Pull request by Doy-lee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,35 +283,49 @@ application.
...
+    ```bash
+    cd loki
+    ```
 
-* **Optional**: if you would like a specific [version/tag](https://github.com/loki-project/loki/tags), do a git checkout for that version. eg. 'v3.0.4'. If you don't care about the version and just want binaries from master, skip this step:
+* If you would like a specific [version/tag](https://github.com/loki-project/loki/tags), do a git checkout for that version. eg. 'v3.0.0'. If you don't care about the version and just want binaries from master, skip this step:
 	
-        git checkout v3.0.4
+    ```bash
+    git checkout v0.14.1.0
```
Comment (msgmaxim) [inline README.md]: v3.0.4?

File: README.md [hunk 2]
```diff
@@ -451,19 +507,25 @@ The produced binaries still link libc dynamically. If the binary is compiled on
...
-        docker run -it -d -v /loki/chain:/root/.loki -v /loki/wallet:/wallet -p 22022:22022 loki
+    ```bash
+    # Build using all available cores
+    docker build -t loki .
+    
+    # or build using a specific number of cores (reduce RAM requirement)
+    docker build --build-arg NPROC=1 -t loki .
+    
+    # either run in foreground
+    docker run -it -v /loki/chain:/root/.bitloki -v /loki/wallet:/wallet -p 18080:18080 loki
```
Comment (msgmaxim) [inline README.md]: Use port 22022? The same a few lines below.

File: README.md (hunk 1 above)
Comment (Doy-lee) [reply README.md]: Fixed to v3.0.6

File: README.md (hunk 2 above)
Comment (Doy-lee) [reply README.md]: Updated ports

**Pull request merged**